### PR TITLE
feat: add swap and pools guides

### PR DESCRIPTION
**Pull request by danielsimao**

### CODE REVIEW

**Comment by danielsimao** [comment]
@nud3l are you able to review this?

File: docs/guides/swap.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+At this stage, you are ready to conduct the swap, but before procceding, there are couple of things that might be of your interest to config or beware of:
+
+#### Price Impact
+
+You can access the price impact of your current swap setup by expanding the element with the current token exchange rate.
+
+![Swap price impact](../_assets/img/guide/swap-page-price-impact.png)
+
+[needs docs link with explanation]
```
Comment (danielsimao) [inline docs/guides/swap.md]: Which link do you suggest here? @nud3l

File: docs/guides/swap.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+#### Slippage
+
+You can access the current slippage and chang it by pressing the top right cog element, which will open a modal where you can customize your slippage.
+
+![Swap slippage](../_assets/img/guide/swap-page-slippage.png)
+
+[needs docs link with explanation]
```
Comment (danielsimao) [inline docs/guides/swap.md]: Which link do you suggest here? @nud3l

File: docs/guides/pools.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+At this stage, you are ready to deposit liquidity, but before procceding, there are couple of things that might be of your interest to config or beware of:
+
+#### Slippage
+
+You can access the current slippage and chang it by pressing the top right cog element, which will open a modal where you can customize your slippage.
+
+![Swap slippage](../_assets/img/guide/swap-page-slippage.png)
+
+[needs docs link with explanation]
```
Comment (danielsimao) [inline docs/guides/pools.md]: Which link do you suggest here? @nud3l

**Comment by nud3l** [comment]
@danielsimao could you check with @gregdhill please for the doc links or details? Thanks!

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (gregdhill) [inline yarn.lock]: Remove this file

File: docs/guides/swap.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+You can access the price impact of your current swap setup by expanding the element with the current token exchange rate.
+
+![Swap price impact](../_assets/img/guide/swap-page-price-impact.png)
+
+[needs docs link with explanation]
+
+#### Slippage
+
+You can access the current slippage and chang it by pressing the top right cog element, which will open a modal where you can customize your slippage.
```
Comment (gregdhill) [inline docs/guides/swap.md:87]:
```suggestion
You can access the current slippage and change it by pressing the top right cog element, which will open a modal where you can customize your slippage.
```

File: docs/guides/pools.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+When you input the token amount, the amounts for the rest will be calculated. Because assets in this pool are in a specific ratio that needs to be preserved Along side the amounts changes, you can see how much of LP token you will receive (i.e KSM-KBTC LP Token).
+
+![Deposit Amounts Liquidity](../_assets/img/guide/pools-page-deposit-amounts.png)
+
+At this stage, you are ready to deposit liquidity, but before procceding, there are couple of things that might be of your interest to config or beware of:
+
+#### Slippage
+
+You can access the current slippage and chang it by pressing the top right cog element, which will open a modal where you can customize your slippage.
```
Comment (gregdhill) [inline docs/guides/pools.md:76]:
```suggestion
You can access the current slippage and change it by pressing the top right cog element, which will open a modal where you can customize your slippage.
```

**Review by gregdhill — COMMENTED** [review]
@danielsimao what docs links do you need? I'm not sure I follow, do you just need to add an explanation what slippage means?

**Comment by danielsimao** [comment]
ready @nud3l @gregdhill. If you merge, could you squash my commits into a single commit? I don't want to pollute our commit history :)

**Review by nud3l — APPROVED** [review]

**Pull request merged**